packages/@aws-cdk/aws-glue-alpha/lib/jobs/pyspark-flex-etl-job.ts

```diff
@@ -46,7 +31,7 @@ export interface PySparkFlexEtlJobProps extends JobProperties {
    *
    * @default - no extra python files specified.
    *
-   * @see `--extra-py-files` in https://docs.aws.amazon.com/glue/latest/dg/aws-glue-programming-etl-glue-arguments.html
```

Comment:
`@see` is going to think the entire line is a link, so we can't have additional qualifiers unfortunately.